### DISCUSSION TITLE
Video tutorial

### DISCUSSION THREAD

**Discussion by Faisalthe01** [KnugiHK/WhatsApp-Chat-Exporter#121] [Ideas]
Any video tutorial on how to use this plugin for iPhone?

**Comment by KnugiHK** [comment]
There is no plan on making official tutorials on the exporter. Perhaps the community could!

**Reply by KnugiHK** [reply]
@connect-mahdi Pinging you in case you are interested!

**Reply by connect-mahdi** [reply]
Thanks @KnugiHK . I'll try first to accomplish an old style modest "text" and "screenshot" tutorial on Android ;-)
But definitely a Video Tutorial might help. I don't own an iPhone, so @Faisalthe01 your use case would be complementary to mine (Linux + Android and if possible Windows + Android)
